body.css

```diff
@@ -0,0 +1,250 @@
+/** 
+  * body.css
+  *
+  * These are the styles for the body of the webpage   
+  *
+  * Index
+  * - badge
```

Comment:
what's the point of this comment?
ditto for all the other comments in css files

Reply:
@mentosfreshmaker This type of comment was written to give the reader a clear sense of what the spreadsheet style properties are used for in relation to the HTML page